### DISCUSSION TITLE
V2 expected timeline

### DISCUSSION THREAD

**Discussion by Yamp** [pydantic/pydantic#5189] [Pydantic V2]
Hello

I am very excited about the announced V2 release. And I actively use both rust and python. Still, from the open sources I can estimate neither the expected release date to prepare my projects for upgrading nor any possibilities for contributions to make this release happen sooner.

Can you please advise me on some ways to closely monitor the existing progress and maybe contribute if I can help?

**Comment by samuelcolvin** [comment]
See https://github.com/pydantic/pydantic/issues/4887
